kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/SessionCreatedListener.java

```diff
@@ -122,6 +127,7 @@ public void run() {
 
             if (!configuration.isEnvironmentInitEnabled() || waitStrategy.await()) {
                 displaySessionStatus(session);
+                afterStartEvent.fire(new AfterStart(session));
```

Comment:
Just curious - why do we need to fire this event instead of directly firing `SessionCreatedEvent`?